### PR TITLE
Document Mbed Studio as a debugging option

### PR DESCRIPTION
**Pull request by LDong-Arm**

### CODE REVIEW

File: docs/debugging-testing/debug/debugging_mbed_os_apps.md [hunk 1]
```diff
@@ -6,7 +6,9 @@ The simplest way to debug your code is to augment your code with [printf](../tut
 
 ## Debugging from an IDE
 
-Keil uVision natively supports debugging Mbed OS applications. To set up uVision, read [Debugging with Keil uVision](../debug-test/keil-uvision.html).
+Mbed Studio and Keil uVision natively support debugging Mbed OS applications:
+* To set up Mbed Studio, read [Debugging with Mbed Studio](../debug-test/mbed-studio.html).
```
Comment (LDong-Arm) [inline docs/debugging-testing/debug/debugging_mbed_os_apps.md]: Will the auto-generated HTML have this name?

Comment (iriark01) [reply docs/debugging-testing/debug/debugging_mbed_os_apps.md]: I don't think we need a new page that serves only as a link to another page. You can just link from here directly to the Mbed Studio site.

Comment (LDong-Arm) [reply docs/debugging-testing/debug/debugging_mbed_os_apps.md]: Updated, please have a look

File: docs/debugging-testing/debug/debugging_mbed_os_apps.md [hunk 2]
```diff
@@ -6,7 +6,9 @@ The simplest way to debug your code is to augment your code with [printf](../tut
 
 ## Debugging from an IDE
 
-Keil uVision natively supports debugging Mbed OS applications. To set up uVision, read [Debugging with Keil uVision](../debug-test/keil-uvision.html).
+Mbed Studio and Keil uVision natively support debugging Mbed OS applications:
+* To set up Mbed Studio, read Mbed Studio's [documentation](https://os.mbed.com/docs/mbed-studio/current/monitor-debug/debugging-with-mbed-studio.html).
```
Comment (iriark01) [inline docs/debugging-testing/debug/debugging_mbed_os_apps.md]:
```suggestion
* To set up Mbed Studio, read [Mbed Studio's documentation](https://os.mbed.com/docs/mbed-studio/current/monitor-debug/debugging-with-mbed-studio.html).
```

Comment (LDong-Arm) [reply docs/debugging-testing/debug/debugging_mbed_os_apps.md]: Done

**Pull request merged**